core/templates/components/lockup/lockup.twig

```diff
@@ -42,45 +42,55 @@
  * - region_attributes: Attributes for each region variable
  */
 #}
-<div{{ attributes }} class="su-lockup {{ modifier_class -}}">
+<div{{ attributes }} class="su-lockup {{ modifier_class }}">
   {#- Cell container #1 -#}
   <div class="su-lockup__cell1">
-    {% if main_link is not empty %}
-    <a href="{{ main_link }}" class="su-lockup__wordmark-link">
+    {% block cell1 %}
+
+    {% if link is not empty %}
+    <a href="{{ link }}" class="su-lockup__wordmark-link">
     {% endif %}
       <span class="su-lockup__wordmark">Stanford</span>
     {% if line4 is not empty %}
       <span class="su-lockup__line4" {{ region_attributes.line4 }}>{{ line4 }}</span>
     {% endif %}
-    {%- if main_link is not empty %}
+    {%- if link is not empty %}
     </a>
     {% endif %}
+    {% endblock %}
   </div>
+
   {#- Cell container #2 -#}
   <div class="su-lockup__cell2">
-    {% if secondary_link is not empty %}
-    <a href="{{ secondary_link }}" class="su-lockup__lines-link">
+    {% block cell2 %}
+    {% if link is not empty %}
+    <a href="{{ link }}" class="su-lockup__lines-link">
     {% endif %}
 
     {% if line1 is not empty %}
-    <span class="su-lockup__line1" {{ region_attributes.line1 }}>{{- line1 }}</span>
+    <span class="su-lockup__line1" {{ region_attributes.line1 }}>{{ line1 }}</span>
     {% endif %}
 
     {% if line2 is not empty %}
-    <span class="su-lockup__line2" {{ region_attributes.line2 }}>{{- line2 }}</span>
+    <span class="su-lockup__line2" {{ region_attributes.line2 }}>{{ line2 }}</span>
     {% endif %}
 
     {% if line3 is not empty %}
-    <span class="su-lockup__line3" {{ region_attributes.line3 }}>{{- line3 }}</span>
+    <span class="su-lockup__line3" {{ region_attributes.line3 }}>{{ line3 }}</span>
     {% endif %}
 
-    {%- if secondary_link is not empty %}
+    {% if link is not empty %}
     </a>
     {% endif %}
+    {% endblock %}
   </div>
 
   {#- Line 5 of Site Name - Optional -#}
-  {%- if line5 is not empty %}
-    <div class="su-lockup__line5 {{ region_attributes.line5 }}">{{ line5 }}</div>
+  {% if line5 is not empty %}
```

Comment:
John's testing found that users treat the lockup as a whole entity, and therefore expect the whole thing to behave as the logo does. I think we need to add the `{{ link }}` to line 5.

Reply:
🤔 This may already be too many links. The same link one after the other is discouraged for accessibility reasons as far as I know. I may have to refactor this so that the whole thing gets wrapped instead. This is going to throw off some of the styles but is likely needed.

Reply:
@sherakama 

Touché (two Sheas).

Can we wrap the `<a>` around the most parent `<div>` in the template (`<div class="su-lockup [modifier class]">`)?

Would that be valid in HTML5 since anchors can wrap [almost] any element except for other anchors and form controls?

**_"the <a> element may be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g. buttons or other links)"._**

If we can do that, I do not think the styling would be impacted.

Reply:
Ok, I think I got it without too much shifting. @josephgknox can you have another review.

Reply:
Brilliant! This looks and works great.